.travis.yml

```diff
@@ -6,6 +6,12 @@
 dist: xenial
 
 language: python
+python:
+  - "3.9.7"
```

Comment:
Is there a way to tell travis 'latest'?

Reply:
Not that I know of.

Reply:
[Python versions available in TravisCI are here.](https://docs.travis-ci.com/user/languages/python/#python-versions)